packages/strapi-helper-plugin/lib/src/components/Select/index.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import ReactSelect, { components } from 'react-select';
+import PropTypes from 'prop-types';
+import { Carret } from '@buffetjs/icons';
+import { useTheme } from 'styled-components';
+
+const DropdownIndicator = props => {
+  const theme = useTheme();
+
+  return (
+    <components.DropdownIndicator {...props}>
+      <Carret fill={theme.main.colors.grey} />
+    </components.DropdownIndicator>
+  );
+};
+
+export const Select = ({ children, onChange, selectedValue, ...props }) => {
+  const theme = useTheme();
+  const childrenArray = React.Children.toArray(children);
+
+  const options = childrenArray.map(child => ({
+    value: child.props.value,
+    label: child.props.children,
+  }));
+
+  const selectedOption = options.find(({ value }) => value === selectedValue);
+
+  return (
+    <ReactSelect
+      options={options}
+      onChange={({ value }) => onChange(value)}
+      components={{ DropdownIndicator }}
+      styles={{
```

Comment:
Can you export de style into a dedicated file, it might be interesting to add some tests to it. 

> You should be able to reuse most of the styles from [this folder](https://github.com/strapi/strapi/tree/master/packages/strapi-admin/admin/src/components/Users/SelectRoles/utils) , it contains both the styles to apply to the ReactSelect and the associated tests. A good improvement of the file would be to use to theme.